static/styles/compose.css

```diff
@@ -307,14 +306,16 @@
         margin-bottom: 4.5px;
         height: 32px;
         white-space: nowrap;
+
+        &.right_edge {
+            margin-right: 10px;
```

Comment:
This switches the margin from 11px to 10px, but that seems better since it's consistent with the padding to the left of the close button.